src/graphics/text.c

```diff
@@ -419,8 +419,9 @@ int text_draw_multiline(const uint8_t *str, int x_offset, int y_offset, int box_
     return y - y_offset;
 }
 
-int text_measure_multiline(const uint8_t *str, int box_width, font_t font)
+int text_measure_multiline(const uint8_t *str, int box_width, font_t font, int *largest_width)
 {
+    *largest_width = 0;
```

Comment:
Strictly speaking `largest_width` is an optional parameter so it can be null. I know we only use this in one place (the tooltips), but let's be safe and add a null-check around accessing it.